spec/factories/order_copy_cards_registration.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
```

Comment:
Do we need all this stuff in the factory - overseas address options, etc?

Reply:
oh shut! No we don't! I forgot to delete XD